tools/trip_generator.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+import random
+import json
+import numpy as np
+
+class TripGenerator:
+    DURATION = ['bardzo krótki', 'krótki', 'średni', 'długi']
+    ACCOMODATION_STANDARD = ['zadowalający', 'dobry', 'wysoki', 'luksusowy']
+    PRICE_RANGE = (200, 1000)
+    TRANSPORTATION = ['samolot', 'prom', 'autobus', 'pociąg']
+    BOARD_BASIS = ['we własnym zakresie', 'w cenie', 'all inclusive']
+    CHILDREN_FRIENDLY = ['tak', 'nie']
+    PETS_FRIENDLY = ['tak', 'nie']
+    TOURIST_DENSITY = ['niski', 'umiarkowany', 'wysoki', 'bardzo wysoki']
+    SOUVERNIR_SHOPS_PER_SQ_KM_RANGE = (0, 50)
+    
+    COUNTRIES = {
```

Comment:
we can add more regions later so it seems a little bit more interesting but it's not important now

Reply:
Yeah, the list is definitely worth extending in the future but it seem like a minor thing